graphql/util.go

```diff
@@ -160,13 +160,26 @@ func GetBaseTaskStatusesFromPatchID(d data.Connector, patchID string) (BaseTaskS
 
 // SchedulePatch schedules a patch. It returns an error and an HTTP status code. In the case of
 // success, it also returns a success message and a version ID.
-func SchedulePatch(ctx context.Context, patchId string, version *model.Version, patchUpdateReq PatchVariantsTasksRequest) (error, int, string, string) {
+func SchedulePatch(ctx context.Context, patchId string, version *model.Version, patchUpdateReq PatchVariantsTasksRequest, parametersModel []*restModel.APIParameter) (error, int, string, string) {
 	var err error
 	p, err := patch.FindOneId(patchId)
 	if err != nil {
 		return errors.Errorf("error loading patch: %s", err), http.StatusInternalServerError, "", ""
 	}
 
+	// parameters cannot be set once the patch has been finalized
+	if parametersModel != nil && len(p.Version) != 0 {
```

Comment:
Say from the CLI I defined a parameter myParam (so it's currently stored for the patch), but then at the UI stage I realized I didn't want it so I deleted it -- will parametersModel be sent as nil or as an empty list? (Or is that to be determined?) We'd want it to be an empty list so that we overwrite what's in patch

Reply:
good point. I'll change it

Reply:
Discussed further offline